### PR TITLE
remove all 3.34 => 3.36 gnome-shell compatibility code

### PR DESCRIPTION
**Pull request by PapyElGringo**

### CODE REVIEW

File: src/layout/main.ts [hunk 1]
```diff
@@ -484,34 +479,30 @@ export class MonitorContainer extends St.Widget {
         });
     }
 
-    allocateHorizontalPanelSpacer(
-        box: Clutter.ActorBox,
-        flags?: Clutter.AllocationFlags
-    ) {
-        AllocatePreferredSize(this.horizontalPanelSpacer, flags);
+    allocateHorizontalPanelSpacer(box: Clutter.ActorBox) {
```
Comment (HalfVoxel) [inline src/layout/main.ts:482]: Parameter not used?

File: src/prefs/prefs-gtk3.ts.old [hunk 2]
```diff
@@ -48,9 +48,7 @@ export function buildGtk3() {
     theme.addSetting('panel-icon-color', WidgetType.BOOLEAN);
     theme.addSetting('taskbar-item-style', WidgetType.COMBO);
     theme.addSetting('surface-opacity', WidgetType.INT);
-    if (!ShellVersionMatch('3.34')) {
-        theme.addSetting('blur-background', WidgetType.BOOLEAN);
```
Comment (HalfVoxel) [inline src/prefs/prefs-gtk3.ts.old:52]: This shouldn't have been removed? Right?

**Review by HalfVoxel — CHANGES_REQUESTED** [review]

File: src/prefs/prefs-gtk3.ts.old (hunk 2 above)
Comment (PapyElGringo) [reply src/prefs/prefs-gtk3.ts.old:52]: indeed

File: src/layout/main.ts (hunk 1 above)
Comment (PapyElGringo) [reply src/layout/main.ts]: I removed the function as it useless

File: src/prefs/prefs-gtk3.ts.old (hunk 2 above)
Comment (PapyElGringo) [reply src/prefs/prefs-gtk3.ts.old:52]: But this file is garbage anyway

**Review by HalfVoxel — APPROVED** [review]

**Pull request merged**